hs_rest_api2/metadata.py

```diff
@@ -59,7 +59,9 @@ def resource_metadata(resource):
     """
     file_with_path = resource.scimeta_path
     istorage = resource.get_irods_storage()
-    metadata_dirty = resource.getAVU('metadata_dirty')
+    metadata_dirty = resource.update_relation_meta()
+    if not metadata_dirty:
+        metadata_dirty = resource.getAVU('metadata_dirty')
```

Comment:
Again, if metadata_dirty AVU is set to True in update_relation_meta() when metadata is updated and dirty, the check here can be simplified to only check AVU unless update_relation_meta() somehow needs to be accessed here?

Reply:
`update_relation_meta()` doesn't set the AVU.

Reply:
I am now setting AVU in `update_relation_meta()` which is needed to update the resourcemetadata.xml file.